custom/icds_reports/static/icds_reports/mobile/css/app.less

```diff
@@ -326,3 +329,40 @@
 .alert-maintenance .close {
   margin-left: 3px;
 }
+
+.text-underline {
+  text-decoration: underline;
+}
+
+.sort-popup {
+  width: 280px;
+  max-height: 350px;
+  overflow: auto;
+  z-index: 1;
```

Comment:
can this be moved to `varaibles.less`? we try to track all z-indices there so it's clear what should sit on top of what